backend/reservations/views/admin.py

```diff
@@ -142,144 +144,112 @@ def guest_search(arr, low, high, x):
-                              email=guest_new['email'],
-                              ticket=guest_new['ticket_code'],
-                              jwt=otp,
-                              room_number = room_num)
-
-
-        logger.debug(f"updated: {existing_ticket}, fields: {existing_ticket[0].name},{existing_ticket[0].ticket},{existing_ticket[0].room_number}")
-
-    else:
-        logger.debug(f'[*] Email exists, ticket exists, room exists, everything matches entry')
+    logger.info(f"[+] Assigned room number: {room.number}")
+    if(email_onboarding):
+        time.sleep(5)
+        if os.environ.get('ROOMBAHT_SEND_MAIL', 'FALSE').lower() == 'true':
+            apppass = os.environ['ROOMBAHT_EMAIL_HOST_PASSWORD']
+            logger.debug(f'[+] Sending invite for guest {guest_new["first_name"]} {guest_new["last_name"]}')
+
+            body_text = f"""
+                BleepBloopBleep, this is the Room Service RoomBaht for Room Swaps letting you know the floors have been cleaned and you have been assigned a room. No bucket or mop needed.
+                
+                After you login below you can view your current room, look at other rooms and send trade requests. This functionality is only available until Monday 11/7 at 5pm PST, so please make sure you are good with what you have or trade early.
+                
+                Goes without saying, but don't forward this email.
+                
+                This is your password, there are many like it but this one is yours. Once you use this password on a device, RoomBaht will remember you, but only on that device.
+                Copy and paste this password. Because let’s face it, no one should trust humans to make passwords:
+                {otp}
+                http://rooms.take3presents.com/login
+                
+                Good Luck, Starfighter.
+
+            """
+
+            send_mail("RoomService RoomBaht",
+                      body_text,
+                      "placement@take3presents.com",
+                      [guest_new["email"]],
+                      auth_user="placement@take3presents.com",
+                      auth_password=apppass,
+                      fail_silently=False,)
 
 
 def create_guest_entries(init_file=""):
-    dr = None
-    new_guests=[]
-    gl = list(Guest.objects.all())
-    guest_sort(gl)
+    guest_rows = []
+    guests_existing = list(Guest.objects.all())
+    guest_sort(guests_existing)
     try:
-        glen = len(gl)-1
+        glen = len(guests_existing)-1
     except TypeError as e:
         logger.warning(f"No existing guests to show ")
         glen=0
 
     with open(init_file, "r") as f1:
-        dr = []
-        for elem in DictReader(f1):
-            dr.append(elem)
+        for row in DictReader(f1):
+            stripd = {k.lstrip().rstrip(): v.lstrip().rstrip() for k, v in row.items() if type(k)==str and type(v)==str}
```

Comment:
I might just yeet this into the `helpers.py` mod heh.

Reply:
Lulz seems legit